.github/workflows/release-create-kpt-functions.yaml

```diff
@@ -9,10 +9,11 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-    - name: Use Node.js 12.x
+    - uses: actions/checkout@v1
+    - name: Use Node.js 14.x
       uses: actions/setup-node@v1
       with:
-        node-version: 12.x
```

Comment:
we should also update the node version in Dockerfile in a future PR

Reply:
Will do this later today.